ui/src/components/navbar.tsx

```diff
@@ -87,6 +91,25 @@ export class Navbar extends Component<any, NavbarState> {
     }
 
     WebSocketService.Instance.getSite();
+
+    this.handleSearchParam = this.handleSearchParam.bind(this);
```

Comment:
This line isn't necessary, you don't need to bind when you use linkEvent.

Reply:
done